web/home.go

```diff
@@ -1,11 +1,37 @@
 package web
 
 import (
+	"fmt"
+	"html/template"
 	"net/http"
-
-	"github.com/gin-gonic/gin"
 )
 
-func homeHandler(c *gin.Context) {
-	c.HTML(http.StatusOK, "home.html.tmpl", gin.H{})
+// Index data is used for the home template
+type Index struct {
+	Title     string
+	Copyright string
+}
+
+func IndexHandler(templates map[string]template.Template) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		data := Index{
```

Comment:
Why do we have the `Title` and `Copyright` here in the `home` handler?
This values are used only in the layout and they shouldn't belong here.
The previous implementation added this items before, so you don't need to have them in every handler code

Reply:
the previous implementation where using some go gin internal. 

I think we need to investigate if we can pass a global struct to all templates as the gin did

Reply:
We built the `addLayoutData` function on top of the following gin struct:
https://github.com/gin-gonic/gin/blob/master/render/html.go#L40-L45

We'd basically need to do the same with the new data structure you're introducing.